CoroutineTemplate/app/build.gradle.kts

```diff
@@ -15,13 +15,20 @@ val keystoreProperties = rootDir.loadGradleProperties("signing.properties")
 
 android {
     signingConfigs {
-        create("config") {
+        create(BuildType.RELEASE) {
             // Remember to edit signing.properties to have the correct info for release build.
-            storeFile = file("../keystore_production")
+            storeFile = file("config/release.keystore")
             storePassword = keystoreProperties.getProperty("KEYSTORE_PASSWORD") as String
             keyPassword = keystoreProperties.getProperty("KEY_PASSWORD") as String
             keyAlias = keystoreProperties.getProperty("KEY_ALIAS") as String
         }
+
+        getByName(BuildType.DEBUG) {
+            storeFile = file("config/debug.keystore")
+            storePassword = "oQ4mL1jY2uX7wD8q"
+            keyAlias = "debug-key-alias"
+            keyPassword = "oQ4mL1jY2uX7wD8q"
```

Comment:
Optional: Move these values to `signing.properties` for consistency

Ex:

```
RELEASE_KEY_ALIAS=ENTER_KEY_ALIAS_NAME
RELEASE_KEYSTORE_PASSWORD=ENTER_KEYSTORE_PASSWORD
RELEASE_KEY_PASSWORD=ENTER_KEY_PASSWORD

DEBUG_KEY_ALIAS=debug-key-alias
DEBUG_KEYSTORE_PASSWORD=oQ4mL1jY2uX7wD8q
DEBUG_KEY_PASSWORD=oQ4mL1jY2uX7wD8q
```

Reply:
@Tuubz I don't think so. Moving them to `signing.properties` would confuse the dev because it's not a thing to change or provide when releasing the app. Debug keystore info is fixed and no need to change 💭